### DISCUSSION TITLE
What is the format of `timestamp` in the report/dag/trace configuration?

### DISCUSSION THREAD

**Discussion by gdevenyi** [nextflow-io/nextflow#4753] [Q&A] [answered]
I'm getting filenames like this:
`report-20240216-44809773.html`

I recognize the first half as the ISO format of date, however, I have no idea of the second half.

Why not a full ISO format?
```bash
$ date --iso=seconds
2024-02-16T12:39:42-05:00
```

**Comment by JosuaCarl** [comment, marked as answer]
The format is: "yyyyMMdd-A". I have no idea why, though... It's kinda bad to read.

The 'A' stands for milliseconds that were passed during this day.
